docs-gen/content/rule_set/branches.md

```diff
@@ -5,30 +5,55 @@ weight: 1
 ---
 
 A branch entry describes a tree branch (or node) containing other branches and
-signals.
+signals.  Branches typically represent either a feature of interest (an object whose properties are being observed or reported), or a node that acts as a parent to a group of related properties.
 
-A branch entry example is given below:
+A branch entry representing a feature of interest is exemplified below:
 
 ```YAML
 Trunk:
   type: branch
-  description: All signals related to the rear trunk
-  aggregate: false
+  identifier: Vehicle.Body.Trunk
+  elementType: FeatureOfInterest
+  instances: ["Front", "Rear"]
+  definition: An enclosed lockable storage area in a sedan, coupe, or convertible separate from the passenger cabin
+  description: Trunk status.
+  comment: A trunk is a luggage compartment in a vehicle.
+           Depending on vehicle, it can be either in the front or back of the vehicle.
+           Some vehicles may have trunks both at the front and at the rear of the vehicle.
+```
+
+A branch entry representing a parent node is exemplified below:
+
+```YAML
+Windshield.Wiping:
+  type: branch
+  elementType: Node
+  description: Windshield wiper signals.
 ```
 
 The following elements are defined:
 
-**`Body.Trunk`**
+**`Trunk:`** or **`Windshield.Wiping:`**
 The list element name defines the dot-notated signal name to the signal.
 Please note that all parental branches included in the name must be defined as
-well.
+well.  In the case of ```Trunk```, it is a child of ```Vehicle.Body``` and thus both ```Vehicle``` and ```Vehicle.Body``` need to be defined.
 
 **```type```**
-The value ```branch``` specifies that this is a branch entry (as
-opposed to a signal entry). This is the default, in case ```type``` is omitted.
+The value ```branch``` specifies that this is a branch entry. This is the default, in case ```type``` is omitted.
+
+**```identifier```** *[optional]* 
+A set of characters that uniquely identifies the branch, usually the full path
+
+**```elementType```** *[optional]* 
+A type that categorizes the branch's role in the vehicle signal domain, or as a Node if the branch exists purely for tree traversal and has no vehicle signal domain signifcance on its own.
```

Comment:
Is there some rule, how this Applies to current VSS, i.e. how to decide what is what? Are "Features of Interest" only things that are currently instantiated and everything else is a "Node", i.e waht would Vehicle.Body.Lights be?

Also what is the difference between `FeatureOfInterest`and `Node`

Reply:
A feature of interest is any physical object whose properties can be observed or reported.  Vehicle.Body.Lights has a property (LightSwitch) that can be reported, so it's a feature of interest.  (I'm not saying it's an ideal one.) There are branches in VSS that simply act as parent nodes to a group of signals.  I just looked and they've mostly been cleaned up :^)  This one remains in vehicle.vspec:

#
# Spatial Acceleration
#
Acceleration:
  type: branch
  description: Spatial acceleration. Axis definitions according to ISO 8855.

Reply:
I think a `featureOfInterest` does not necessarily have to be a physical object. It can also refer to a specific aspect or characteristic of something, such as the `charging` function of an electrical vehicle.

Reply:
Yes, thank you.  In fact, in our internal model we have a feature of interest called "chargingSession," which is a process.  I'm not quite sold on functions, and in fact we modeled all charging signals without a thing called "charging."  But I think about functions in the philosophical sense as something that a feature of interest is intended to achieve.  That BFO book I recommended explains all that.